SP5WWP/m17-packet/m17-packet-encode.c

```diff
@@ -21,7 +28,7 @@ uint16_t num_bytes=0;                                       //number of bytes in
 uint8_t fname[128]={'\0'};                                  //output file
 
 FILE* fp;
-float full_packet[6912+88];                                 //full packet, symbols as floats - (40+40+32*40+40+40)/1000*4800
+float full_packet[1920*37];                                 //full packet, samples as floats - (40+40+32*40+40+40)/1000*4800
```

Comment:
Isn't this excessive? The largest packet is 40+40+32*40+40+40ms long (preamble, LSF, 32 frames, ending frame, EOT). Add 5 symbols for RRC flushing (assume RRC filter span of 8 symbols, 8/2+1=5). That's 3456+5=3461 symbols. In samples, that's 34610 (assuming sps=10). 1920*37=71040 is a much larger number. Am I missing something?

Reply:
Pfft, excessive, what's that lol. 

Its been so long since I made these changes, I don't recall if this was necessary or not. I thought I did it to house the upsampled symbols or something, but I may have changed how I did that and forgot to set this back. I'll have to look at it when I have a chance to see if I can jog my memory on it, and if necessary, revert it back to its original value.

Reply:
Original value is probably too large. 34610 should be OK.

Reply:
My math is wrong. 36 "frames" max (incl. preamble, LSF, EoT), 192 symbols each: `[36*192*10]`.